versions.tf

```
@@ -6,12 +6,12 @@ terraform {
   # major version currently being used.  This practice will help us
   # avoid unwelcome surprises.
   required_providers {
-    # Version 3.38.0 of the Terraform AWS provider is the first
-    # version to support default tags.
-    # https://www.hashicorp.com/blog/default-tags-in-the-terraform-aws-provider
+    # Version 3.75.0 of the Terraform AWS provider backports the changes to how
+    # AWS S3 resources are structured from the 4.0 release.
```

I would word this a little differently:
```suggestion
    # Version 3.75.0 of the Terraform AWS provider backports from the 4.0 release
    # the changes to how AWS S3 resources are structured.
```

---

Do we have any other alternatives? I am not a fan of how your suggestion reads.

---

It was just a suggestion.  You can ignore it.

---

Right, but if you aren't a fan of the language I'd rather come to a common ground on improving the language. @dav3r Do you have any thoughts?

---

I had the thought to see what's used in [cisagov/cyhy_amis](https://github.com/cisagov/cyhy_amis) to be consistent but it looks like I completely forgot that it's [where I got the language](https://github.com/cisagov/cyhy_amis/blob/5430bf305a75c0abe648cc8728077af57f0bf88a/terraform/versions.tf#L10-L12) in the first place. 😑

---

What is there is fine.  What you had makes sense, it just seemed a little awkward to me so I made a suggestion.